docs/extensibility/configmap-sections.md

```diff
@@ -1,26 +1,21 @@
-# Resource-based extensions
+# Config Map sections
```

Comment:
Not sure about this change... The name and title of this file were this on purpose, as we assume we might want to create different config map extensions in the future that are not resource-based. Sure for now it's just this one, but...